stdlib/CRC32c/test/runtests.jl

```diff
@@ -77,3 +77,7 @@ end
 crc32c_sw(io::IO, crc::UInt32=0x00000000) = crc32c_sw(io, typemax(Int64), crc)
 test_crc32c(crc32c)
 test_crc32c(crc32c_sw)
+
+@testset "Docstrings" begin
+    @test Docs.undocumented_names(CRC32c) == []
```

Comment:
`isempty` feels nicer to me at least.

Reply:
I thought `==` provided more information on a test failure, but now I see that `@test` will print out the `isempty` argument on a failure too. 

Will fix.

Reply:
Fixed.